jobsdb/jobsdb_utils.go

```diff
@@ -192,16 +192,16 @@ func constructParameterJSONQuery(table string, parameterFilters []ParameterFilte
 	for _, parameter := range parameterFilters {
 		allKeyValues = append(allKeyValues, fmt.Sprintf(`"%s":"%s"`, parameter.Name, parameter.Value))
 		if parameter.Optional {
-			opNullConditions = append(opNullConditions, fmt.Sprintf(`"%s".parameters -> '%s' IS NULL`, table, parameter.Name))
```

Comment:
iirc the double quotes were added to support -ve indices in table_names(`gw_jobs_-1`). Were the double-quotes causing an issue..?

Reply:
It was causing the test to fail, reverted it, will make the changes in test if it fails again.